graylog2-web-interface/src/components/extractors/ExtractorExampleMessage.jsx

```diff
@@ -9,25 +12,33 @@ class ExtractorExampleMessage extends React.Component {
     onExampleLoad: PropTypes.func,
   };
 
+  static defaultProps = {
+    example: '',
+    onExampleLoad: () => {},
```

Comment:
I am seeing this defaultProp `() => {}` for the first time. It looks like it is a common practice in the project. Are we adding this default prop to prevent an error, if `onExampleLoad` is not defined?